source/documentation/common/security/proc-Applying_the_PCI_DSS_Security_Profile_RHV.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// doc-Administration_Guide/common/security/assembly-Securing_Red_Hat_Virtualization.adoc
+// THIS MODULE IS RHV ONLY.
+
+:_content-type: PROCEDURE
+[id='applying-the-pci-dss-profile-rhv_{context}']
+
+= Applying the PCI-DSS profile in {virt-product-shortname} hosts and the standalone Manager
+
+When installing {hypervisor-shortname}, you can select the DISA STIG profile with the GUI installer, which is the profile provided by {enterprise-linux-shortname} 8.
+
+.Procedure
+
+. On the *Installation Summary* screen, select *Security Policy*.
```

Comment:
I suggest saying "In the Installation Summary screen"

Reply:
Done, thanks